package.json

```diff
@@ -40,6 +40,8 @@
     "eslint": "1.9.0",
     "eslint-plugin-react": "3.6.3",
     "express": "4.13.3",
+    "react-transform-catch-errors": "1.0.0",
+    "react-transform-hmr": "1.0.1",
```

Comment:
By the way it does not support babel6. As well as https://github.com/gaearon/babel-plugin-react-transform.
Not an issue here, but worth knowing.